sdk/devcenter/azure-developer-devcenter/samples/create_devbox_sample.py

```diff
@@ -31,10 +31,12 @@
 from azure.identity import DefaultAzureCredential
 from azure.core.exceptions import HttpResponseError
 
+
 def get_project_name(LOG, client):
     projects = list(client.projects.list_by_dev_center(top=1))
     return projects[0].name
 
+
 def main():
     logging.basicConfig(level=logging.DEBUG)
```

Comment:
One more thing, I know this PR didnt add these lines but we should removing these lines that are enabling debug level logging by default. I think this was done because on line 47 an error is logged for this samples instead of just raising an error.

Reply:
removed debug level logging